Makefile

```diff
@@ -55,6 +55,7 @@ lint:
 	fi
 	
 	for PKG in $(PACKAGES); do golint -set_exit_status $$PKG || exit 1; done;
+	go vet
```

Comment:
maybe can add new .PHONY:vet like:

``` sh
.PHONY: vet
vet:
 $(GO) vet $(PACKAGES)
```

Reply:
and add to https://github.com/swaggo/swag/blob/889705a8db7763121543be8c314d836ebf040d42/.travis.yml#L19-L23

Reply:
ok